jenkins/v2/cortx-ha.spec

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+Name: <RPM_NAME>
+Version: %{version}
+Release:  %{build_num}_git%{git_rev}%{?dist}
+Summary: HA Tools
+License: Seagate Proprietary
+URL: https://github.com/Seagate/cortx-ha
+Source0: <RPM_NAME>-%{version}.tar.gz
+#TODO: Dependency on Hare rpm
+%define debug_package %{nil}
+
+%description
+HA Tools
+
+%prep
+%setup -n cortx
+# Nothing to do here
+
+%build
+
+%install
+rm -rf ${RPM_BUILD_ROOT}
+mkdir -p ${RPM_BUILD_ROOT}<HA_PATH> ${RPM_BUILD_ROOT}<HA_PATH>/ha/bin
+cp -rp . ${RPM_BUILD_ROOT}<HA_PATH>
+
+mkdir -p ${RPM_BUILD_ROOT}/usr/lib/ocf/resource.d/cortx/
+mv resource/{dispatch,lnet,sspl} \
+   ${RPM_BUILD_ROOT}/usr/lib/ocf/resource.d/cortx/
+rm -rf resource
```

Comment:
`rm -rf resource`

we should not remove anything from packaging

Reply:
This was present in v1 only so I did remove the same from v2 as we don't need this in v2.